docs/playground/playground-context.asciidoc

```diff
@@ -0,0 +1,64 @@
+[role="xpack"]
+[[playground-context]]
+== Optimize model context
+
+preview::[]
+
+// Variable (attribute) definition 
+:x:                    Playground 
+
+Context is the information you provide to the LLM, to optimize the relevance of your query results.
+Without additional context, an LLM will generate results solely based on its training data.
+In {x}, this additional context is the information contained in your {es} indices.
+
+There are a few ways to optimize this context for better results.
+Some adjustments can be made directly in the {x} UI.
+Others require refining your indexing strategy, and potentially reindexing your data.
+
+[float]
+[[playground-context-ui]]
+== Edit context in UI
+
+Use the *Edit context* button in the {x} UI to adjust the number of documents and fields sent to the LLM.
+
+If you're hitting context length limits, try the following:
+
+* Limit the number of documents retrieved
+* Pick a field with less tokens, reducing the context length
+
+[float]
+[[playground-context-index]]
+== Other context optimizations
+
+This section covers additional context optimizations that you won't be able to make directly in the UI.
+
+[float]
+[[playground-context-index-chunking]]
+=== Chunking large documents
+
+If you're working with large fields, you may need to adjust your indexing strategy.
+Consider breaking your documents into smaller chunks, such as sentences or paragraphs.
+
+If you don't yet have a chunking strategy, start by chunking your documents into passages.
+
+Otherwise, consider updating your chunking strategy, for example, from sentence based to paragraph based chunking.
+
+Refer to https://www.elastic.co/search-labs/blog/chunking-via-ingest-pipelines[Chunking large documents via ingest pipelines (blog post)] for inspiration.
```

Comment:
this isn't a great link because nested KNN doesn't work in playground today. the notebooks im putting up in here https://github.com/elastic/elasticsearch-labs/pull/232 will be good however. Worth just replacing the urls for that.

[JSON](https://github.com/elastic/elasticsearch-labs/tree/main/notebooks/ingestion-and-chunking/json-chunking-ingest.ipynb)
[PDF](https://github.com/elastic/elasticsearch-labs/tree/main/notebooks/ingestion-and-chunking/pdf-chunking-ingest.ipynb)
[Website](https://github.com/elastic/elasticsearch-labs/tree/main/notebooks/ingestion-and-chunking/website-chunking-ingest.ipynb)